rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.4.4-release-notes.adoc

```diff
@@ -0,0 +1,31 @@
+= RPA Recorder 6.4.4 Release Notes
+
+RPA Recorder enables you to record a business process for automation.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== October 9, 2023
+
+// == Compatibility
+include::partial$release-notes/rn-headings.adoc[tag=compatibility,leveloffset=+1]
+
+[%header%autowidth.spread]
+|===
+|Application |Version
+|RPA Builder | 6.4.8 and later
+|===
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="2a,1a"]
+|===
+| Issue Resolution | ID
+|The CefSharp.Common library is upgraded to version 116.0.23 to address reported security vulnerabilities.| W-14234524
```

Comment:
Sorry, there was a "0" missing in our Confluence page.
The correct version is 116.0.230 , not .160.0.23